### PR TITLE
Add Persian (FA) translation

### LINKED ISSUE #424: Add Persian translation json file
Add Persian language translation. Original author: `@MR5857` in Telegram. Here is the translation itself https://gist.github.com/ro31337/ed182538dbebd62f9f99bb3315738e83

Unfortunately, I have no skills with editing right-to-left text, and I don't know how GitHub and Telegram handles it (I assume it handles OK).

So in this story do the following:

* Check translation by comparing keys to English: https://github.com/ro31337/libretaxi/blob/master/locales/en.json
* Pay attention to `passenger-contact-new-number.message` and `passenger-contact-driver-price.message` - there is information that translation can be incorrect because GitHub doesn't support (?) right-to-left text (?).
* Make sure strings containing `%s`, `{{something}}` spelled correctly and there are no unicode characters between them (otherwise string interpolation will fail).
* Make sure _all_ icons are in place. For example, when I type "🚗 ماشین" into google translate, I get the following:

image

It seems right, in Persian icon is at the beginning (at the end visually). However, in json file it's at the end (at the beginning visually).

In Telegram it also seems right (when you paste "🚗 ماشین" without quotes, you get exactly what is expected):

image

The translation above can be technically correct, just attention from someone familiar with Git and Persian language is required.

So In this story:

* Do all the checks written above
* Add Persian translation file and create PR (you can create PR to the `master` branch, it's OK for this story)
* In PR message **verbosely explain** what has been done, give as much information as it is possible.

This issue is high priority one and Persian-speaking community is looking forward to use the bot. Please help the world to be a better place by working on this issue.

### PR DESCRIPTION
**Pull request by Alir3z4**
Fixes #424

### CODE REVIEW

File: locales/fa.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	"passenger-request-destination.provide_destination": "مقصد شما؟ لطفاً آدرس دقیق خیابان را وارد کنید (تا جهت یاب جی‌پی‌اس آن را پیدا کند) یا مقصد خود را بنویسید",
+	"order-submitted.order_submitted": "⏳ در جستجوی راننده ها...اگر راننده خود را پیدا کردید، کلید زیر را فشار دهید 👇",
+	"order-submitted.cancel": "پایان جستجو",
+	"order-submitted.error_incorrect_input": "انتخاب نامعتبر، لطفاً دوباره تلاش کیند",
+	"order-cancelled.order_cancelled": "👌 تشکر، پایان جستجوی راننده‌ها",
+	"driver-select-vehicle-type.select": "نوع ماشینی که به عنوان تاکسی از آن استفاده می کنید را انتخاب کنید",
+	"driver-select-vehicle-type.motorbike": "🏍 اسکوتر / موتورسیلکت",
+	"driver-select-vehicle-type.car": "🚗 ماشین",
+	"driver-select-vehicle-type.error_only_known_type": "انتخاب نامعتبر، لطفاً ماشین یا اسکوتر / موتورسیلکت را انتخاب کنید",
+	"driver-explain-checkins.text": "تقریباً تمام!، چند نکته: تلگرام پشتیبانی لحظه‌ای موقعیت جی‌پی‌اس ندارد، بنابراین هر زمان موقعیت خود را تغییر می دهید، دوباره وارد شوید یا اگر آماده دریافت درخواست‌های جدید هستید هر یک ساعت دوباره وارد شوید. این به ما کمک مییکند تا مسافران اطراف شما را پیدا کنیم. روی /"بعدی/" کلیک کنید ",
```
Comment (xesina) [inline locales/fa.json]: Please fix errors :)

Comment (Alir3z4) [inline locales/fa.json]: @xesina Thanks for having a look at this.
Got the errors fixed.

Please let me know if you found any other issues.

Comment (xesina) [inline locales/fa.json]: @Alir3z4 I've tested a fully working version of the bot with your translations. Nice job :)

**Comment by ro31337** [comment]
Thanks for contribution, I'm merging to temporary branch from where I'll integrate translation. Will be added to the app today.

**Pull request merged**